projects/1.20/assets/rubinated-nether/rubinated_nether/lang/zh_cn.json

```
@@ -50,11 +57,63 @@
   "block.rubinated_nether.freezer": "冷冻器",
   "menu.rubinated_nether.freezer": "冷冻器",
   "gui.rubinated_nether.recipebook.toggleRecipes.freezable": "仅显示可冷冻",
-
   "gui.rubinated_nether.jei.freezer": "冷冻",
   "gui.rubinated_nether.jei.fuel": "制冷剂",
-
   "gui.rubinated_nether.emi.freezes": "冷冻%s个物品",
   "emi.category.rubinated_nether.freezing": "冷冻",
-  "emi.category.rubinated_nether.freezer_fuel": "制冷剂"
+  "emi.category.rubinated_nether.freezer_fuel": "制冷剂",
+
+  "item.rubinated_nether.shimmer_disc": "镶红宝石音乐唱片",
+  "item.rubinated_nether.shimmer_disc.desc": "Quizzly - Shimmer",
+
+  "trim_material.rubinated_nether.ruby": "红宝石质",
+
+  "config.rubinated_nether.creative_tab": "启用创造标签页",
+  "config.rubinated_nether.creative_tab.desc": "启用模组的创造标签页；无论启用与否，模组中物品会一直出现在原版的标签页中\n§e需要重启游戏",
+
+  "config.rubinated_nether.netherite_smithing_template_recipe": "下界合金模板红宝石化",
+  "config.rubinated_nether.netherite_smithing_template_recipe.desc": "启用模组自定义的下界合金锻造模板配方\n§e需要重启游戏",
+
+  "config.rubinated_nether.chandelier": "灯台",
+  "config.rubinated_nether.chandelier_damage_multiplier": "伤害倍数",
+  "config.rubinated_nether.chandelier_damage_multiplier.desc": "灯台坠落时伤害增加的速率",
+  "config.rubinated_nether.chandelier_max_damage": "最大伤害",
+  "config.rubinated_nether.chandelier_max_damage.desc": "灯台坠落能造成的最大伤害",
+
+  "config.rubinated_nether.brazier_effect_range": "效果范围",
+  "config.rubinated_nether.brazier_effect_range.desc": "火盆效果的作用范围，以格计",
+  "config.rubinated_nether.brazier_effect_duration": "效果持续时间",
+  "config.rubinated_nether.brazier_effect_duration.desc": "火盆效果的持续时间，以秒计",
+  "config.rubinated_nether.brazier_effect_particles": "效果粒子",
+  "config.rubinated_nether.brazier_effect_particles.desc": "火盆启用信标效果粒子\n起效前可能需要先等待已有的效果消失",

```

原文：Enable beacon effect particles for brazier.\nYou may need to wait for the effect to ware off for this to apply


```suggestion
  "config.rubinated_nether.brazier_effect_particles.desc": "对火盆启用信标效果粒子\n可能待已有的效果消失后才能起效",
```